internal/oauth2/lazy_token.go

```diff
@@ -69,7 +71,10 @@ func (lt *LazyToken) ForceRefresh(ctx context.Context) (string, error) {
 }
 
 // ForceRefresh forces the acquisition of a new access token by clearing the current AccessToken and calling Token().
-func (lt *LazyToken) requestToken() (*tokenResponse, error) {
+func (lt *LazyToken) requestToken(ctx context.Context) (*tokenResponse, error) {
+	span, _ := strc.Start(ctx, "oauth2 token request")
+	defer span.End()
+
```

Comment:
If this one turns to be noisy, let's remove it. This is only called when token expires (for each pod).